doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,18 +187,272 @@ if transceiver is not present:
+          CmisManagerTask ->> APPL_DB: PORT_TABLE:<lport>.NPU_SI_SETTINGS_SYNC_STATUS = <br> NPU_SI_SETTINGS_NOTIFIED
+          CmisManagerTask ->> CmisManagerTask : Transition CMIS SM to CMIS_STATE_NPU_SI_SETTINGS_WAIT
+        end
+        activate OA
+        APPL_DB -->> OA: Notify NPU SI settings for ports
+        Note over OA: Disable admin status<br>setPortSerdesAttribute
+        OA ->> APPL_DB: PORT_TABLE:<lport>.NPU_SI_SETTINGS_SYNC_STATUS = NPU_SI_SETTINGS_DONE
+        APPL_DB --> CmisManagerTask: PORT_TABLE:<lport>.NPU_SI_SETTINGS_SYNC_STATUS = NPU_SI_SETTINGS_DONE
+        Note over OA: initHostTxReadyState
+        deactivate OA
+        CmisManagerTask ->> CmisManagerTask : Transition CMIS SM to CMIS_STATE_DP_INIT
+    end
+```
+
+## XCVRD termination during syncd/swss/orchagent crash
+
+The below sequence diagram captures the termination of XCVRD during syncd/swss/orchagent crash.
+<br> supervisord will respawn XCVRD after termination as xcvrd is killed using SIGABRT signal
+
+```mermaid
+sequenceDiagram
+    participant OA
+    participant APPL_DB as APPL_DB@asic_n
+    participant XCVRDMT as XCVRD main thread
+    participant CmisManagerTask
+    participant DomInfoUpdateTask
+    participant SfpStateUpdateTask
+
+    activate OA
+    activate XCVRDMT
+    activate CmisManagerTask
+    activate DomInfoUpdateTask
+    activate SfpStateUpdateTask
+    OA -x OA: Crashes while handling a routine
+    deactivate OA
+    OA ->> APPL_DB : DEL PORT_TABLE
+
+    XCVRDMT -x APPL_DB : XCVRD main thread proecesses DEL event of APPL_DB PORT_TABLE
+    Note over XCVRDMT: generate_sigabrt = True
+    alt If threads > 0 are dead
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGKILL
+    end
+    XCVRDMT -x CmisManagerTask : Stop CmisManagerTask
+    deactivate CmisManagerTask
+    XCVRDMT -x DomInfoUpdateTask : Stop DomInfoUpdateTask
+    deactivate DomInfoUpdateTask
+    XCVRDMT -x SfpStateUpdateTask : Stop SfpStateUpdateTask
+    deactivate SfpStateUpdateTask
+    Note over XCVRDMT : deinit()
+    alt self.sfp_error_event.is_set()
+        XCVRDMT -x XCVRDMT : sys.exit(SFP_SYSTEM_ERROR)
+    else if generate_sigabrt is True
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGABRT
+
+    else
+        XCVRDMT -x XCVRDMT : Graceful exit
+    end
+    deactivate XCVRDMT
+```
+
+## Test plan and expectation
+| Event          | APPL_DB_<asic_n> cleared | Xcvrd restarted | NPU SI settings renotify | NPU_SI_SETTINGS_SYNC_STATUS value on xcvrd boot-up for initialized transceiver | CMIS re-init triggered | Link flap |
+| -------------- | ------------------------ | --------------- | ------------------------ | ------------------------------------------------------------------------------ | ---------------------- | --------- |
+| Xcvrd restart  | N                        | Y               | N                        | NPU_SI_SETTINGS_DONE                                                           | N                      | N         |
```

Comment:
To this table, add transceiver removal and transceiver insertion events depicting how all these columns would be handled

Reply:
I have addressed this now.